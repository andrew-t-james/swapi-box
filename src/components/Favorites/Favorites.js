import React from 'react';
import PropTypes from 'prop-types';

const Favorites = () => {
  return (
    <div>
      Favoritesd
    </div>
  );
};

Favorites.propTypes = {

};

export default Favorites;
