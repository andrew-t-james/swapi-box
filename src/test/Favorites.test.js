import React from 'react';
import { shallow } from 'enzyme';
import Favorites from '../components/Favorites/Favorites';

describe('Favorites', () => {
  let wrapper;

  beforeEach(() => wrapper = shallow(<Favorites />));

  afterEach(() => wrapper.unmount());

  test('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
