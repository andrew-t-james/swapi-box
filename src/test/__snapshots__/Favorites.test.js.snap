// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Favorites should match snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Favorites />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": "Favoritesd",
    },
    "ref": null,
    "rendered": "Favoritesd",
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "Favoritesd",
      },
      "ref": null,
      "rendered": "Favoritesd",
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
    "disableLifecycleMethods": true,
  },
}
`;
